codecov.yml

```diff
@@ -3,5 +3,6 @@ coverage:
   ignore:
     - third_party/.*
     - bench/.*
+    - test/.*
```

Comment:
doing this brings our codecoverage from 85% down to 79% :(

Reply:
this is probably the kind of change that should be discussed with the whole team. **I reverted this one** as codecov check is not marked as "required"(didn't notice it before)
Though it makes total sense to me: we should mix test code coverage with business-logic code coverage